### ISSUE TITLE
cordova-plugin-console causes build error iOS on PGB 

### ISSUE THREAD

**Issue by MulderWH** [mendix/hybrid-app-base#24]
Hi, I cannot create the iOS app on PGB using the latest 1.7.2 version. When you check the log on PGB you will find the error: duplicate symbol CDVLogger

This error is related to cordova-plugin-console. You need to remove the plugin from the config.xml.mustache. See [https://forums.adobe.com/message/10099469#10099469](https://forums.adobe.com/message/10099469#10099469) 

Can you confirm?

**Comment by KevinVlaanderen** [comment]
It seems that plugin should indeed have been removed. I've pushed a new version without it. Could you try running 'npm update' (from your mobile app project folder) and then try to build a new package, to confirm the fix?

Thanks!

**Comment by MulderWH** [comment]
I cannot confirm the fix. The npm update gives throws an error, see screendump:
![image](https://user-images.githubusercontent.com/8004408/35345347-b39f0a56-012f-11e8-83ec-aa46a908e038.png)

**Comment by KevinVlaanderen** [comment]
It seems the issue was related to the recently added local phonegap dependency. I've pushed new versions of hybrid-app-base and hybrid-app-template, where this local dependency is removed. Phonegap should now be installed globally, and it should match the cli version configured in the config.xml (e.g. npm install -g phonegap@7.1.0).

Could you perhaps give it another go? You'll need a fresh hybrid app package; npm update does not suffice in this case.

**Comment by MulderWH** [comment]
The build is created successfully. I have downloaded a new package from sprintr, and verified the version 1.4.0 in the package.json. Then run 'npm install' and the following error is shown:
![image](https://user-images.githubusercontent.com/8004408/35614927-f4b30e66-0670-11e8-820e-d2b4ea299b77.png)
The fix this I run 'npm install -g phonegap@7.1.0', and then 'npm install' Hope this helps.

**Comment by KevinVlaanderen** [comment]
That's indeed as expected. We removed Phonegap as a local dependency, because that caused issues when running npm commands. More specifically, one of Phonegap's transitive dependencies is npm 2, which is old and interferes with newer npm versions.
The implication is that the phonegap package should from now on be installed globally again. The message you're seeing was added as part of the latest release. I'll update the release notes to reflect this clearly as well.

Thank you for the feedback!
